gpkit/keydict.py

```diff
@@ -149,6 +149,11 @@ class KeySet(KeyDict):
     "KeySets are KeyDicts without values, serving only to filter and map keys"
     collapse_arrays = False
 
+    def add(self, *args):
+        "Adds all args to the keyset"
+        for arg in args:
+            self[arg] = None
```

Comment:
python's `set.add` takes exactly one argument. I think we should match that interface here. https://docs.python.org/2/library/sets.html#sets.Set